test/server/modules/form/api.spec.js

```diff
@@ -82,7 +94,32 @@ describe('Form', () => {
 			});
 		});
 
+		it('should customise form for teamtrial theme', () => {
+			return tester.get({ path: '/form/teamtrial' })
+				.then(({ status, headers, $ }) => {
+					expect(status).to.equal(200);
+					expect(headers['cache-control']).to.equal('max-age=0, no-cache, must-revalidate');
+					expect(headers['surrogate-control']).to.equal('max-age=0, no-cache, must-revalidate');
+
+					expect($('header#site-navigation').length).to.equal(1);
```

Comment:
Should we add these to the others too, to make sure the header footer is turned off?